### DISCUSSION TITLE
How to prevent screen burn

### DISCUSSION THREAD

**Discussion by frehberg** [slint-ui/slint#3549] [General]
How to prevent with Slint a screen burn (LCD/TFT) in industrial application with display/HMI always on?

If any, the desktop screen saver/lock will be disabled.
Monitoring data (a few temperature values) must be displayed all time.
There should be no  floating animation distracting people.

Any idea how to prevent screen burn?
Maybe, would it be sufficient to switch to night mode during idle time?

**Comment by ogoffart** [comment]
I don't really know the answer to that since it depends on what screen and things is used.

Maybe one possibility would be to have a slow animation to move the graphs by a few pixel over a period of a few minutes:
 [Example](https://slintpad.com/?snippet=export+component+Demo+%7B%0A++width%3A+300px%3B%0A++height%3A+300px%3B%0A++Image+%7B%0A++++source%3A+%40image-url%28%22https%3A%2F%2Fslint.dev%2Flogo%2Fslint-logo-full-light.svg%22%29%3B%0A++++width%3A+250px%3B%0A++++x%3A+%28parent.width+-+self.width%29%2F2+%2B+15px+*+cos%28animation-tick%28%29*1turn%2F90s%29%3B%0A++++y%3A+%28parent.height+-+self.height%29%2F2+%2B+20px+*+sin%28animation-tick%28%29*1turn%2F70s%29%3B%0A++%7D%0A%7D%0A)
